website/docs/r/stream_instance.html.markdown

```diff
@@ -50,4 +50,5 @@ You can import stream instance resource using the project ID and instance name,
 $ terraform import mongodbatlas_stream_instance.test 650972848269185c55f40ca1-InstanceName
 ```
 
-To learn more, see: [MongoDB Atlas API - Stream Instance](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Streams/operation/createStreamInstance) Documentation.
+To learn more, see: [MongoDB Atlas API - Stream Instance](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Streams/operation/createStreamInstance) Documentation. 
+The [Terraform Provider Examples Section](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/atlas-streams/README.md) also contains details on the overall support for Atlas Streams in Terraform.
```

Comment:
@Zuhairahmed FYI I included a link to the github examples section from the terraform docs, I believe this will help give it more visibility.

Reply:
perfect this is best practice to have both these links in our docs for new features